web3/auto/http.py

```diff
@@ -0,0 +1,16 @@
+from web3 import (
+    HTTPProvider,
+    Web3
+)
+
+
+def http():
+
+    w3 = Web3(HTTPProvider('http://localhost:8545'))
```

Comment:
It's nice to be able to configure this type of thing using environment variables.  This can be done something like the following.

```python
w3 = Web3(HTTPProvider(
    os.environ.get('WEB3_HTTP_PROVIDER_URI', 'http://localhost:8545')
))
```

This allows for this functionality to be used beyond just `localhost`.